subtests/docker_cli/systemd_in_container/test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Test systemd in containers
+#
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 IMAGE-NAME"
+    exit 1
+fi
+image="$1"
+
+# Exit status. Default 0 (success), but set to 1 on any subtest failure
+rc=0
+
+# Main code. Run a command whose output is expected to be a single line,
+# and the first field is a security context (user:role:type:level); verify
+# that command generates exactly one line of output; extract the <type>
+# and <level> fields; and check against a list of one or more expected
+# values. Assume that there's no overlap between valid types & ranges.
+check_output() {
```

Comment:
I hope to add many more tests like this for systemd in the future, this is just to get us started.

Is the container name an issue? We can make it a parameter to the function.